components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5541,23 +5541,29 @@ private Attribute getSchemaAttributes(ExternalClaim scimClaim, LocalClaim mapped
                                           boolean isExtensionAttr) {
 
         String name = scimClaim.getClaimURI();
+        String claimDielectURI = scimClaim.getClaimDialectURI();
+        boolean isCustomScemaAttr = claimDielectURI.equalsIgnoreCase(getCustomSchemaURI());
```

Comment:
Don't we need null and empty check here ?

Reply:
I think we don't since string equal check functions accepts null values.

Reply:
Used StringUtils in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/459/commits/b5eaab3e1944238b0c0282a38f332f22509a18f5